package.json

```diff
@@ -24,7 +24,7 @@
     "cy:open": "cypress open",
     "test": "npm run cy:start-server-and-test",
     "cy:start-server-and-test": "start-server-and-test serve :5000 cy:run",
-    "cy:run": "cypress run",
+    "cy:run": "cypress run --record --key 0d5ef033-250c-4367-a396-40c9b0668b18",
```

Comment:
Is this a secret?

Reply:
Yes and no but you're right, I'm working to hide it 😉